JsonBodyListener.php

```diff
@@ -41,6 +48,11 @@ public function onKernelRequest(GetResponseEvent $event)
                 throw new BadRequestHttpException('Invalid ' . $format . ' message received');
             }
 
+            $jsonSchema = $request->get('_jsonSchema');
```

Comment:
No, it's more something to help the developper checking that the output respect a contract (the documented schema output).

I'm pretty sure this schema should be applied in functional tests scenario output to ensure the contract is not broken at any time and that the documentation keeps in sync with the features & effectively returned outputs.